package.json

```diff
@@ -23,18 +32,19 @@
     "url": "https://github.com/mendixlabs/rich-text/issues"
   },
   "devDependencies": {
-    "@types/classnames": "^2.2.0",
-    "@types/core-js": "0.9.36",
-    "@types/enzyme": "^2.8.2",
+    "@types/classnames": "^2.2.1",
+    "@types/core-js": "^0.9.42",
+    "@types/enzyme": "^2.8.6",
     "@types/jasmine": "^2.5.53",
     "@types/karma": "^0.13.36",
     "@types/quill": "0.0.31",
-    "@types/react": "^15.0.38",
-    "@types/react-dom": "^15.5.1",
+    "@types/react": "^16.0.2",
```

Comment:
type should be 16. as the web modeler is not supporting it. risking to use react 16 features